applications/emails.py

```diff
@@ -31,11 +35,13 @@ def create_confirmation_email(application, request):
 def create_lastreminder_email(application):
     c = {
         'name': application.user.get_full_name,
+        'type': application.user.get_type_display(),
         # We need to make sure to redirect HTTP to HTTPS in production
         'confirm_url': 'http://%s%s' % (settings.HACKATHON_DOMAIN,
                                         reverse('confirm_app', kwargs={'id': application.uuid_str})),
         'cancel_url': 'http://%s%s' % (settings.HACKATHON_DOMAIN,
                                        reverse('cancel_app', kwargs={'id': application.uuid_str})),
+        'is_hacker': application.user.is_hacker(),
```

Comment:
Are you sending last reminders to sponsors?

Reply:
No, sponsors are confirmed on apply. Those are for mentors and volunteers.

Reply:
Then why is there a is_sponsor variable?

Reply:
I cannot see what do you mean, there isn't a is_sponsor variable there